Makefile

```diff
@@ -362,93 +351,34 @@ switch-kind:
 switch-tanzu:
 	@ echo "tanzu" > hack/deployer/config/provider
 
-
-
 #################################
 ##  --    Docker images    --  ##
 #################################
 
-BUILD_PLATFORM ?= "linux/amd64,linux/arm64"	
-
-publish-operator-multiarch-image:
-	@ .buildkite/scripts/build/is_published.sh $(OPERATOR_IMAGE) $(BUILD_PLATFORM) \
-	|| $(MAKE) docker-multiarch-build
-
-docker-multiarch-build: go-generate generate-config-file
-	@ hack/docker.sh -l -m $(OPERATOR_IMAGE)
-	docker buildx build . \
-		--progress=plain \
-		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
-		--build-arg GO_TAGS='$(GO_TAGS)' \
-		--build-arg VERSION='$(VERSION)' \
-		--platform $(BUILD_PLATFORM) \
-		-t $(OPERATOR_IMAGE) \
-		--push
-ifeq ($(PUBLISH_IMAGE_DOCKERHUB),true)
-	@ $(MAKE) docker-multiarch-build-dockerhub
-endif
-ifeq ($(PUBLISH_IMAGE_UBI),true)
-	@ $(MAKE) docker-multiarch-build-ubi
-endif
-
-docker-multiarch-build-dockerhub:
-	docker buildx build . \
-		--progress=plain \
-		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
-		--build-arg GO_TAGS='$(GO_TAGS)' \
-		--build-arg VERSION='$(VERSION)' \
-		--platform $(BUILD_PLATFORM) \
-		-t $(OPERATOR_DOCKERHUB_IMAGE) \
-		--push
-
-docker-multiarch-build-ubi:
+# build amd64 image for dev purposes
+BUILD_PLATFORM ?= "linux/amd64"
+docker-push-operator:
 	docker buildx build . \
+	 	-f build/Dockerfile \
 		--progress=plain \
 		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
 		--build-arg GO_TAGS='$(GO_TAGS)' \
 		--build-arg VERSION='$(VERSION)' \
 		--platform $(BUILD_PLATFORM) \
-		-f Dockerfile.ubi \
-		-t $(OPERATOR_IMAGE_UBI) \
-		--push
-
-publish-operator-image:
-	@ docker pull $(OPERATOR_IMAGE) \
-	&& echo "OK: image $(OPERATOR_IMAGE) already published" \
-	|| $(MAKE) docker-build docker-push
-
-docker-build: go-generate generate-config-file
-	DOCKER_BUILDKIT=1 docker build . \
-		--progress=plain \
-		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
-		--build-arg GO_TAGS='$(GO_TAGS)' \
-		--build-arg VERSION='$(VERSION)' \
+		--push \
 		-t $(OPERATOR_IMAGE)
 
-docker-push:
-	@ hack/docker.sh -l -p $(OPERATOR_IMAGE)
+drivah-generate-operator:
+	@ build/gen-drivah.toml.sh
 
-buildah-login:
-	@ buildah login \
-		--username="$(shell vault read -field=username $(VAULT_ROOT_PATH)/docker-registry-elastic)" \
-		--password="$(shell vault read -field=password $(VAULT_ROOT_PATH)/docker-registry-elastic)" \
-		$(REGISTRY)
-
-buildah-operator-image: go-generate generate-config-file buildah-login
-	buildah bud \
-		--isolation=chroot --storage-driver=vfs \
-		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
-		--build-arg GO_TAGS='$(GO_TAGS)' \
-		--build-arg VERSION='$(VERSION)' \
-		--platform $(BUILD_PLATFORM) \
-		-t $(OPERATOR_IMAGE) .
-	buildah push \
-		--storage-driver=vfs \
-		$(OPERATOR_IMAGE)
+# standard way to build operator image(s) using drivah
+export BUILD_FLAVORS ?= dev
+drivah-build-operator: drivah-generate
+	drivah build ./build
```

Comment:
Something I ran into when building locally through `drivah`

```
er:  > [builder 8/9] COPY build/license.key /license.key:
⚠️  Docker: ------
⚠️  Docker: tmph_dbdkyw:24
⚠️  Docker: --------------------
⚠️  Docker:   22 |     COPY pkg/ pkg/
⚠️  Docker:   23 |     COPY cmd/ cmd/
⚠️  Docker:   24 | >>> COPY $LICENSE_PUBKEY_PATH /license.key
⚠️  Docker:   25 |     
⚠️  Docker:   26 |     # Build
⚠️  Docker: --------------------
⚠️  Docker: ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 300e217c-0ac1-4f9f-b498-54e1f5adee6b::b118lvsqa95wl97f8fqsagh5m: "/build/license.key": not found
```

Somehow the build context does not contain the `license.key`, when I build manually with docker from the generated Dockerfile it seems to work though. The file is present in `build/license.key`

Reply:
`build/license.key` is created by `gen-drivah.toml.sh`. You could try to `set -x` in it to see what happens.

If `BUILD_FLAVORS=dev`, we create a fake license, which will not be used but the Dockerfile wants to copy a file.
https://github.com/elastic/cloud-on-k8s/blob/067621980a8af4cb860b6929316fa275fa828842/build/gen-drivah.toml.sh#L75
else it's read from Vault:
https://github.com/elastic/cloud-on-k8s/blob/067621980a8af4cb860b6929316fa275fa828842/build/gen-drivah.toml.sh#L94-L99

If you have a Vault error, you should see an error : `Error reading secret/..: Error making API request`.

Reply:
The fake license file has been generated fine. But for some reason it cannot be copied in the when building the image.